src/de/schildbach/pte/PlProvider.java

```diff
@@ -74,6 +77,13 @@ protected void addCustomReplaces(final StringReplaceReader reader) {
     protected Product normalizeType(final String type) {
         final String ucType = type.toUpperCase();
 
+		if ("SKM".equals(ucType)) // SKM Trojmiasto in Gdansk Metropolitan Area
```

Comment:
Indentation mismatch - the code seems to use spaces, whereas your contribution is tabs